pkgs/development/perl-modules/generic/default.nix

```diff
@@ -20,12 +20,16 @@ perl.stdenv.mkDerivation (
     # authors to skip certain tests (or include certain tests) when
     # the results are not being monitored by a human being."
     AUTOMATED_TESTING = true;
+
+    # current directory (".") is removed from @INC in Perl 5.26 but many old libs rely on it
+    # https://metacpan.org/pod/release/XSAWYERX/perl-5.26.0/pod/perldelta.pod#Removal-of-the-current-directory-%28%22.%22%29-from-@INC
+    PERL_USE_UNSAFE_INC = perl.stdenv.lib.optionalString (perl.stdenv.lib.versionAtLeast perl.version "5.26") "1";
   }
   //
   attrs
   //
   {
-    name = "perl-" + name;
+    name = "perl${perl.version}-${name}";
```

Comment:
mass rebuild caused by this changing in names modeled after Python's naming (```python2.6-lxml```, ```python2.7-lxml```, ...)